target/linux/ramips/image/mt7620.mk

```diff
@@ -802,6 +802,16 @@ define Device/ohyeah_oy-0001
 endef
 TARGET_DEVICES += ohyeah_oy-0001
 
+define Device/phicomm_k2
+  SOC := mt7620a
+  IMAGE_SIZE := 7552k
+  DEVICE_VENDOR := Phicomm
+  DEVICE_MODEL := K2
+  DEVICE_VARIANT:= v22.5+
```

Comment:
What's that?

Reply:
Before official firmware version v22.5, bootloader load kernel from 0x50000. But new version Phicomm K2 has an additional partition named "permanent_config", and bootloader is not compatible with PSG1218 anymore. I don't have a perfect idea to distinguish this difference so I add a DEVICE_VARIANT. But if we can add some additional illustration on openwrt wiki, I think this one can be removed.

Reply:
So, we need a phicomm_k2-v22.5-plus and a phicomm_k2-v22.4-minus then?

Reply:
Looks a little strange. I found there are only four main software versions, v21.4.x.x, v22.4.x.x, v22.5.x.x and v22.6.x.x. 
And this devices was end of support by official site for a long time. Which of the following two options do you think is more appropriate? I can't decide.
phicomm_k2-v22.5-v22.6 & phicomm_k2-v21.4-v22.4
phicomm_k2-v22.5-plus & phicomm_k2-v22.4-minus

Reply:
Should I rename psg1218 to phicomm_k2-v22.4-minus now?

Reply:
My point was mainly that we need to find a "good" naming scheme here. I'm not sure what would be "good" in this case yet, though.

Reply:
It seems there are only 5 main software versions, v21.4.x.x, v22.3.x.x, v22.4.x.x, v22.5.x.x and v22.6.x.x. And this devices was end of support by official site for a long time. I think phicomm_k2-v22.5-v22.6 & phicomm_k2-v21.4-v22.4 can be a better choice.